app/models/cloud_network.rb

```diff
@@ -55,6 +55,57 @@ def self.tenant_id_clause_format(tenant_ids)
     ["((tenants.id IN (?) OR cloud_networks.shared IS TRUE OR cloud_networks.external_facing IS TRUE) AND ext_management_systems.tenant_mapping_enabled IS TRUE) OR ext_management_systems.tenant_mapping_enabled IS FALSE OR ext_management_systems.tenant_mapping_enabled IS NULL", tenant_ids]
   end
 
+  def update_cloud_network_queue(userid, options = {})
```

Comment:
Lets add a "feature" for this to `SupportsFeatureMixin` for the new operations https://github.com/ManageIQ/manageiq/blob/master/app/models/mixins/supports_feature_mixin.rb#L60

Reply:
Nevermind I see you using create/update/delete features https://github.com/gubbe505/manageiq-providers-nsxt/blob/master/app/models/manageiq/providers/nsxt/network_manager/cloud_network.rb#L5